client/src/features/sessionsV2/SessionStartPage.tsx

```diff
@@ -567,3 +604,46 @@ export default function SessionStartPage() {
 
   return <StartSessionFromLauncher launcher={launcher} project={project} />;
 }
+
+function ShowContainerImageError({
+  error,
+  launcherName,
+  steps,
+  containerImage,
+  projectUrl,
+}: {
+  error: FetchBaseQueryError | SerializedError;
+  launcherName: string;
+  steps: StepsProgressBar[];
+  containerImage: string;
+  projectUrl: string;
+}) {
+  if (!("status" in error)) {
+    return false;
```

Comment:
You should not return `false` for a component.

```suggestion
    return null;
```